tests/ignite/metrics/test_metrics_lambda.py

```diff
@@ -478,13 +474,20 @@ def Fbeta(r, p, beta):
         data = list(range(n_iters))
         state = evaluator.run(data, max_epochs=1)
 
+        y_pred = idist.all_gather(y_pred)
+        y_true = idist.all_gather(y_true)
+
+        y_true = y_true.reshape(n_iters * idist.get_world_size(), batch_size).cpu().numpy()
```

Comment:
Why do you need to reshape tensors here ? Below we are doing `y_true.ravel()` and `y_pred.reshape(-1, n_classes)`